### PR TITLE
fix: Fixed the problem that the cssText attribute cannot be obtained …

### PR DESCRIPTION
**Pull request by aique127**
##### Checklist

- [x] `npm test` passes
- [x] tests are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change
fix: Fixed the problem that the cssText attribute cannot be obtained when @keyfram has a -webkit prefix style in IE11

ie11环境下，当@keyframe带有-webkit-私有前缀时，使用cssRules获取不到cssText导致报错: 找不到成员。
例如使用autoprefixe插件自动添加前缀
```css
// 转换前
@keyframes donghua {
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
}
// autoprefixe插件转换后
@keyframes donghua {
    from {
        transform: rotate(0deg);
        -webkit-transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
        -webkit-transform: rotate(360deg);
    }
}
```
**希望能在qiankun代码里进行错误捕获。防止获取cssText报错**

- close https://github.com/umijs/qiankun/issues/2527

### CODE REVIEW

**Comment by wangdaodao** [comment]
期待早日合并：）

File: src/sandbox/patchers/css.ts [hunk 1]
```diff
@@ -102,7 +102,12 @@ export class ScopedCSS {
           css += this.ruleSupport(rule as CSSSupportsRule, prefix);
           break;
         default:
-          css += `${rule.cssText}`;
+          try {
```
Comment (kuitos) [inline src/sandbox/patchers/css.ts:105]: 不建议这里 try catch，可以参考这个方案修复 https://github.com/gregnb/react-to-print/pull/99/files

**Review by kuitos — CHANGES_REQUESTED** [review]

File: src/sandbox/patchers/css.ts (hunk 1 above)
Comment (aique127) [reply src/sandbox/patchers/css.ts]: 已经重新提交了：）

**Review by kuitos — APPROVED** [review]
Nice work~

**Pull request merged**